README.md

```diff
@@ -24,6 +28,32 @@ If you have Go developer tools installed, you can install it with `go install`:
 $ go install github.com/stateful/runme@latest
 ```
 
+## Commands
+### Help
+```sh {interactive=false}
+  runme help
+```
+
+### List 
+```sh
+  runme list
+```
+
+### Print
+```sh {interactive=false}
+  runme print hello-world
+```
+
+### Run selected command, Example: Update brew
+```sh
+runme run update-brew
```

Comment:
I guess indentations in the commands above like `Help`, `List`, and `Print` are unnecessary. Let's keep it consistent.